src/screens/discover/components/DiscoverHome.js

```diff
@@ -27,28 +27,29 @@ import config from '@/model/config';
 export default function DiscoverHome() {
   const { network } = useAccountSettings();
   const accountAsset = useAccountAsset(ETH_ADDRESS);
-  const profilesEnabled = useExperimentalFlag(PROFILES);
+  const profilesEnabledLocalFlag = useExperimentalFlag(PROFILES);
+  const profilesEnabledRemoteFlag = config.profiles_enabled;
   const hardwareWalletsEnabled = useExperimentalFlag(HARDWARE_WALLETS);
   const opRewardsLocalFlag = useExperimentalFlag(OP_REWARDS);
   const opRewardsRemoteFlag = config.op_rewards_enabled;
   const testNetwork = isTestnetNetwork(network);
+  const isProfilesEnabled =
+    profilesEnabledLocalFlag && profilesEnabledRemoteFlag;
 
   return (
     <Inset top="20px" bottom={{ custom: 150 }}>
       <Stack space="20px">
         <Inset horizontal="20px">
-          {profilesEnabled &&
-          !testNetwork &&
-          !isZero(accountAsset.balance.amount) ? (
```

Comment:
you can remove `isZero` and `accountAsset` and `useAccountAsse` usage

Reply:
oops